fetch.bs

```diff
@@ -7185,6 +7198,12 @@ constructor steps are:
    <dt><a for=request>priority</a>
    <dd><var>request</var>'s <a for=request>priority</a>.
 
+   <dt><a for=request>render-blocking</a>
+   <dd><var>request</var>'s <a for=request>render-blocking</a>.
```

Comment:
I saw your comment about adding these, but given that internal priority is copied over and therefore non-null it's not clear to me when these would actually end up doing anything. Unless we decided to expose them to service workers, but we should probably do this at that point then.

Reply:
Works for me.  Removed. Implementations can still copy them over if they are needed anyway.